### DISCUSSION TITLE
The Working cursor is not the actual one in macOS

### DISCUSSION THREAD

**Discussion by iWajdi** [antiden/macOS-cursors-for-Windows#46] [General]
the title is pretty much self explanatory the working cursor is not the same as in macOS, personally that's annoying  because the quality you give is amazing, but that's the only thing missing.

Found this thumbnail to show it to you

![image](https://user-images.githubusercontent.com/77938896/154294046-4f2b8604-fca4-4ba7-8d19-ea734272d4ac.png)

**Comment by antiden** [comment]
Hi,

I don't understand you, what you want?

**Comment by iWajdi** [comment]
this cursor is not the actual one in macOS 
![image](https://user-images.githubusercontent.com/77938896/154491607-d3319b08-89bf-471d-afc2-f4606f93f688.png)

this is the real one
![image](https://user-images.githubusercontent.com/77938896/154492237-d271d061-d372-4b11-af3e-bd0fa1520c13.png)
could you find a way to change it, please?

**Comment by antiden** [comment]
Same as in MacOS Monterey. For you exit don't use these cursors =)

**Comment by iWajdi** [comment]
Dude, im not here to hate on your work, I just told you these look different, Ive been using macos for quite somrtimes and I know the difference. Dont take this the other way. I appreciate your effort and everything you did, I just said they are not the same.

**Comment by antiden** [comment]
@iWajdi Converted for discussion, i think this is not issue. Need more info, what specifically is wrong?

**Reply by iWajdi** [reply]
I literally just lesrned about this discussion feature, my bad, should have started a discussion instead of an issue. One second I'll gather some info and Reply back here

**Comment by iWajdi** [comment]
![d9zt7rh-d07517fa-3634-4b9a-bdaf-c22fbeb383b5](https://user-images.githubusercontent.com/77938896/154797101-d25a504a-2b2c-45ee-8ff9-07382ee09c5d.jpg)
I keep posting stuff from other people, but this is what macos cursors should look like, there is not blue beachball just rainbow ones, but every pack found on the internet looks bad, and low quality. Thats why I wanted to discuss this with you because you are the knly one who has a pack that looks smooth and clean

**Reply by antiden** [reply]
As I already wrote, these cursors are a copy of MacOS Monterey cursors. If you want El Capitan cursors, hm, I'll think about it.

**Reply by iWajdi** [reply]
No no, no need for the capitain ones, its just ro demonstrate what I want, could you please send me the raw cursor files I dont have access to macos anymore.

**Reply by antiden** [reply]
Unfortunately, these are my cursors that I drew from scratch and cannot provide.

**Reply by iWajdi** [reply]
I thought you extracted them from Macos Installation, that is totally understandable
